SharedPackages/UIComponents/Sources/UIComponents/Shared/FeatureGridView.swift

```diff
@@ -0,0 +1,284 @@
+                    borderWidth: borderWidth
+                )
+            case .fixed:
+                FeatureFixedGrid(
+                    features: features,
+                    columns: columns,
+                    spacing: spacing,
+                    cellMinHeight: cellMinHeight ?? LayoutConstants.defaultCellMinHeight,
+                    borderWidth: borderWidth
+                )
+            }
+        }
+    }
+}
+
+// MARK: - Individual Feature Card
+
+struct FeatureCardView: View {
+    let feature: FeatureGridItem
+    let minHeight: CGFloat?
+    let borderWidth: CGFloat
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: LayoutConstants.contentSpacing) {
+            // Icon at the top
+            HStack {
+                iconPlaceholder
+                Spacer()
+            }
+
+            // Text content below the icon
+            VStack(alignment: .leading, spacing: LayoutConstants.textSpacing) {
+                Text(feature.title)
+                    .font(titleFont)
+                    .foregroundColor(Color(designSystemColor: .textPrimary))
+                    .fixedSize(horizontal: false, vertical: true)
+
+                Text(feature.description)
+                    .font(bodyFont)
+                    .foregroundColor(Color(designSystemColor: .textSecondary))
+                    .fixedSize(horizontal: false, vertical: true)
+                    .lineLimit(nil)
+                    .multilineTextAlignment(.leading)
+            }
+
+            if minHeight != nil {
+                Spacer(minLength: 0)
+            }
+        }
+        .padding(LayoutConstants.cardPadding)
+        .frame(minHeight: minHeight, maxHeight: minHeight != nil ? .infinity : nil)
+        .background(Color(designSystemColor: .surface))
+        .cornerRadius(LayoutConstants.cornerRadius)
+        .overlay(
+            RoundedRectangle(cornerRadius: LayoutConstants.cornerRadius)
+                .stroke(Color(designSystemColor: .lines), lineWidth: borderWidth)
+        )
+    }
+
+    // Platform-specific font handling - matching original platform implementations
+    private var titleFont: Font {
+        #if os(iOS)
+        return Font(UIFont.daxFootnoteSemibold())
+        #else
+        return .title3
+        #endif
+    }
+
+    private var bodyFont: Font {
+        #if os(iOS)
+        return Font(UIFont.daxFootnoteRegular())
+        #else
+        return .body
+        #endif
+    }
+
+    @ViewBuilder
+    private var iconPlaceholder: some View {
+        Circle()
+            .fill(Color(designSystemColor: .surface))
+            .frame(width: LayoutConstants.iconContainerSize, height: LayoutConstants.iconContainerSize)
+            .overlay(
+                Circle()
+                    .stroke(Color(designSystemColor: .lines), lineWidth: borderWidth)
+            )
+            .overlay(
+                Group {
+                    if let iconImage = feature.iconImage {
```

Comment:
This image logic feels slightly repetitive to me. I understand the need to work around `NS` vs `UI` images, but then we have the name-based init.